tensorrt_llm/_torch/modules/fused_moe/quantization.py

```
@@ -1084,7 +1086,7 @@ def create_weights(self, module: torch.nn.Module):
 
         cell_div = lambda x, y: (x + y - 1) // y
```

Ruff (0.16.0)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `cell_div` as a `def`

(E731)